### DISCUSSION TITLE
Can not ping. How to check the connection?

### DISCUSSION THREAD

**Discussion by hellohawaii2** [bubuntux/nordlynx#117] [Q&A]
I run the docker first, and then use `docker exec -it image_name /bin/sh` and then `ping www.google`. 

I got `ping: bad address 'www.google.com'`. However, in the docker, I can see `Connected! \(ᵔᵕᵔ)/ `. I can not ping with an IP address, either.

**Comment by sapshah** [comment]
Did you find an answer? I am facing the same issue

**Comment by mikedorsi** [comment]
I'm having the same issues as well. I see a connected message in the logs but I cannot ping anything outside the local network via bash console.

**Comment by Ansem93** [comment]
yeah, not even traceroute is working
PING 8.8.8.8 (8.8.8.8): 56 data bytes
ping: sendto: Operation not permitted
root@# traceroute 8.8.8.8
traceroute to 8.8.8.8 (8.8.8.8), 30 hops max, 46 byte packets
 1traceroute: sendto: Operation not permitted

**Comment by nathanbeddes** [comment]
I use this:

```
curl ifconfig.io
```

It only returns the current IP address, but it's enough to determine if there's a connection or not.

**Comment by bennevans** [comment]
If I add a DNS to the ALLOW_LIST environment variable, I'm able to ping it. Is there any way to allow all dnss?
